sdks/java/core/src/main/java/org/apache/beam/sdk/options/PipelineOptionsFactory.java

```diff
@@ -171,6 +183,13 @@ public static Builder fromArgs(String... args) {
     return new Builder().fromArgs(args);
   }
 
+  /**
+   * @return a builder instance enabling you to build pipeline options.
+   */
+  public static Builder builder() {
```

Comment:
Unused and it looks like the other initialization methods handle it.

Reply:
Yes and no. Idea is to provide a clean builder entry point. Others are just shortcuts but way less relevant now you have multiple (2) entry points you can combine in the order you want to handle overrides accordingly your need. Alternative is to drop the fromArgs and provide a argsToMap utility but it breaks the compatibility.

Reply:
In that case, just a little nit that they should be `addMap` and `addArgs`. This way it is clear that they keep the existing options and add new ones.

Reply:
Hmm, do we break the API then? (on a personnal note/feeling: a builder with withers sounds as good)